variables.tf

```
@@ -1,3 +1,8 @@
+variable "additional_route_tables" {
+  description = "Controls if route table should be added to subnet)"
+  type        = bool
+  default     = false
+}
```

Understanding the problem you are trying to solve, I don't think `additional_route_tables` is a good name. Also your description ends with an unmatched `)`, the order of the fields is wrong and you didn't separate the variable from the others using a blank line...

I suggest to rename the variable like this:

```suggestion
variable "shared_public_route_table" {
  type        = bool
  description = "Determines weather to use a single route table for all public networks"
  default     = true
}
```

Also note the order of `type`, `description` and `default`. If you look at all the other vars in this file you will notice they all use the same order, so please keep the code consistent and use the same order instead of just doing something random.

After renaming this variable please also make sure to place it in the correct (alphabetical) location and that variables are separated by a space.

---

@svanharmelen why would this be restricted only to public subnets ? the same pattern is there in private also where private subnets have single RT

---

note also that the suggestion offered above is flipping the logic. a value of true will keep the current behaviour of the module and carries through to all other suggestions below.

---

updated